src/components/Loading/Loading.tsx

```diff
@@ -4,7 +4,7 @@ import LoadingGif from '@/assets/loading.gif';
 
 import './Loading.css';
 
-export default function Loading() {
+export default function Loading(): JSX.Element {
```

Comment:
Investigate what is the difference between **`React.FC<Props>`** and **`JSX.Element`**.